package.yaml

```diff
@@ -15,7 +15,7 @@ homepage: http://haskellstack.org
 custom-setup:
   dependencies:
   - base >=4.10 && < 5
-  - Cabal
+  - Cabal >= 2.4
```

Comment:
I don't think this constraint is needed.  We avoid putting any versions constraints in `package.yaml` since the `stack.yaml` will ensure the right version is used (and we use something like of `--pvp-bounds=both` when uploading to Hackage).